src/software/utils/main_sfmSplitReconstructed.cpp

```diff
@@ -0,0 +1,103 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2017 AliceVision contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include <aliceVision/sfmData/SfMData.hpp>
+#include <aliceVision/sfmDataIO/sfmDataIO.hpp>
+#include <aliceVision/system/Logger.hpp>
+#include <aliceVision/system/cmdline.hpp>
```

Comment:
need to rebase on develop and replace with `<aliceVision/cmdline/cmdline.hpp>` (and add the corresponding link in the CMakeLists.txt)